bin/oref0-upload-profile.js

```diff
@@ -60,7 +60,7 @@ if (!module.parent) {
 
     if (apisecret.length != 40) {
         var shasum = crypto.createHash('sha1');
-        shasum.update(apisecret);
+        shasum.update(String(apisecret));
```

Comment:
I accidentally pulled this in, but it fixes a bug that occurs when a user has a numeric Nightscout authentication code.